lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -102,7 +102,10 @@ def report_line_items
 
     def report_variant_overrides
       @report_variant_overrides ||=
-        Reports::VariantOverrides.new(order_permissions.visible_line_items).indexed
+        Reports::VariantOverrides.new(
+          line_items: order_permissions.visible_line_items,
+          distributor_ids: report_line_items.orders.result.pluck('DISTINCT distributor_id'),
```

Comment:
`report_line_items.orders.result` is an `ActiveRecord::Relation` here, so I think in this case `#select` will be more efficient than `#pluck` for building the `distributor_ids` part of the subquery, e.g:
```ruby
distributor_ids: report_line_items.orders.result.select(:distributor_id),
```

Reply:
Done :)